### PR TITLE
Add Power Platform Workshop Page

### PR DESCRIPTION
**Pull request by MalteRei**

### CODE REVIEW

File: static/Events/PPWorkshop.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+            </section>
+
+            <section id="DescriptionSection">
+                <div id="Description" class="sectionDiv">
+                    <div id="overview">
+                        <h4>Date</h4>
+                        <p>22.01.2021</p>
+                        <br/>
+                        <h4>Time</h4>
+                        <p>04:00 pm - 05:00 pm CEST</p>
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: We are currently in CET due to wintertime.

File: static/Events/PPWorkshop.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                        <br/>
+                        <h4>Place</h4>
+                        <p>Teams Online Meeting</p>
+                        <p><a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_ODIzNDlhZmQtMTY2Mi00NThlLWJhYzctODk1MzU5YTMwMmI0%40thread.v2/0?context=%7b%22Tid%22%3a%2284c31ca0-ac3b-4eae-ad11-519d80233e6f%22%2c%22Oid%22%3a%227c85ffcf-b716-4202-b5e0-22df34d099f4%22%7d" style="color: var(--color-mcc-blue)">Click here to join</a></p>
+                    </div>
+                    <div id="endText">
+                        <h2>Low-code / no-code development for all!</h2>
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: Titles look better if new words are written with a capital letter: Low-Code / No-Code Development for All!

Furthermore, a "slash" does not look good. Perhaps, better choose one of the possibilities.

File: static/Events/PPWorkshop.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                        <p>Lee Zuckett, Program Manager for the Power Platform, will teach us how to build no-code / low-code solutions.</p>
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: Lee Zuckett, **experienced** Program Manager for the Power Platform, will teach **you** how to build low-code / no-code solutions.

Again, perhaps choose one of the possibilities. Personally, I prefer low-code as it fits better. Having the MDSD approach does not mean not to code at all. It is just abstracted.

File: static/Events/PPWorkshop.html [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+                        <h2>The next wave in software development has nothing to do with coding.</h2>
```
Comment (spethso) [inline static/Events/PPWorkshop.html:77]: Is this text from Lee? Again, we abstract the coding and generate code from models. I would disagree that this approach has nothing to do with coding. 😉 However, I can accept the text as it is used as an interest catcher.

File: static/Events/PPWorkshop.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+                        <p>
+                            According to Microsoft, 500 million new apps will be built in the next 5 years. This is more than the number of apps built in the last 40 years.
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: According to Microsoft, 500 million new apps will be built in the next 5 years. This is **even** more than the number of apps built in the last 40 years.

File: static/Events/PPWorkshop.html [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+                    <div id="endText">
+                        <h2>Low-code / no-code development for all!</h2>
+                        <p>Lee Zuckett, Program Manager for the Power Platform, will teach us how to build no-code / low-code solutions.</p>
+                        <h2>The next wave in software development has nothing to do with coding.</h2>
+                        <p>
+                            According to Microsoft, 500 million new apps will be built in the next 5 years. This is more than the number of apps built in the last 40 years. 
+                            <br>
+                            This number of apps will not be built by traditional developers.
+                            These apps will be built by Nurses, Architects, Biologists, Visionaries, ..., [insert your profession].
+                            Join us to learn more about no-code / low-code development.
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: Same comment as above. If you keep both options, please keep the order consistent.

File: static/Events/PPWorkshop.html [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+                    </div>
+                    <div id="endText">
+                        <h2>Low-code / no-code development for all!</h2>
+                        <p>Lee Zuckett, Program Manager for the Power Platform, will teach us how to build no-code / low-code solutions.</p>
+                        <h2>The next wave in software development has nothing to do with coding.</h2>
+                        <p>
+                            According to Microsoft, 500 million new apps will be built in the next 5 years. This is more than the number of apps built in the last 40 years. 
+                            <br>
+                            This number of apps will not be built by traditional developers.
+                            These apps will be built by Nurses, Architects, Biologists, Visionaries, ..., [insert your profession].
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: This number of apps will not be built by traditional developers anymore but by the experts in their respective domains themselves, i.e. Nurses, Architects, Biologists, Visionaries, ..., [insert your profession].

File: static/Events/PPWorkshop.html [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+                            Join us to learn more about no-code / low-code development.
+                            <br><br>
+                            Agenda:
+                            <ul>
+                                <li>
+                                    Introduction Low-Code / No-code
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: See above

File: static/Events/PPWorkshop.html [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+                                </li>
+                                <li>
+                                    Introduction Power Platform / Power Apps
```
Comment (spethso) [inline static/Events/PPWorkshop.html]: Introduction **into the** Power Plattform and Power Apps

File: static/Events/PPWorkshop.html [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+                        <p>Lee Zuckett, Program Manager for the Power Platform, will teach us how to build no-code / low-code solutions.</p>
+                        <h2>The next wave in software development has nothing to do with coding.</h2>
+                        <p>
+                            According to Microsoft, 500 million new apps will be built in the next 5 years. This is more than the number of apps built in the last 40 years. 
+                            <br>
+                            This number of apps will not be built by traditional developers.
+                            These apps will be built by Nurses, Architects, Biologists, Visionaries, ..., [insert your profession].
+                            Join us to learn more about no-code / low-code development.
+                            <br><br>
+                            Agenda:
```
Comment (spethso) [inline static/Events/PPWorkshop.html:83]: Perhaps, we can make the whole agenda a bit bigger in size

File: static/events.html [hunk 11]
```diff
@@ -71,6 +71,28 @@ <h5>All Events</h5>
...
+                                <p>ONLINE EVENT</p>
+                                <p>ONLINE</p>
+                            </div>
+
+                            <div class="eventTitle mccBlue">
+                                <p>Power Platform Workshop</p>
+                            </div>
+
+                            <div class="eventIntroText">
+                                <p>Low-code / no-code development for all!</p>
```
Comment (spethso) [inline static/events.html]: See comments in the specific event page.

**Review by spethso — CHANGES_REQUESTED** [review]
In addition to the specific comments, in my opinion, the background colour is too black. Perhaps, a dark grey looks better.

File: static/Events/PPWorkshop.html (hunk 4 above)
Comment (MalteRei) [reply static/Events/PPWorkshop.html:77]: I wrote the abstract. Indeed my goal is to have a catcher.

File: static/Events/PPWorkshop.html (hunk 10 above)
Comment (MalteRei) [reply static/Events/PPWorkshop.html:83]: I get the point. I don't want to fragment the design tho. This is what Brando produced. It is better if not everyone changes the layout.

**Comment by MalteRei** [comment]
> In addition to the specific comments, in my opinion, the background colour is too black. Perhaps, a dark grey looks better.

I am going to ignore this because it has to do with the entire design and is not specific to this event.

**Comment by MalteRei** [comment]
Thank you @spethso for the valuable feedback.

**Pull request merged**